selfdrive/car/fw_versions.py

```diff
@@ -252,6 +252,11 @@ def get_fw_versions(logcan, sendcan, query_brand=None, extra=None, timeout=0.1,
   for addr in tqdm(addrs, disable=not progress):
     for addr_chunk in chunks(addr):
       for brand, r in requests:
+        # Skip queries based off presence of external panda
+        aux_panda_query = r.bus > 3
+        if aux_panda_query != aux_panda:
+          continue
```

Comment:
make this generic for any number of pandas

Reply:
Will change behavior to querying up to n pandas. So now it will run the internal panda queries as well since we don't know if we want to run queries on panda 3 or 1 and 2 as well. Unless we can always asume the last panda will be the one fingerprinting which I don't think we can?

Reply:
limited the queries to the buses available with all pandas